src/main/java/com/red5pro/cluster/streams/Ingest.java

```diff
@@ -0,0 +1,53 @@
+package com.red5pro.cluster.streams;
+/**
+ * Defines an RTMP path for publishing clients.
+ * Port is assumed to be rtmp for RTC and RTSP clients when resolving instances.
+ * @author Andy Shaules
+ *
+ */
+public class Ingest {
+	
+	public static int SBR = 0;	
+	public static int MBR_MAIN = 1;
+	
```

Comment:
I'm not clear on how an RTC stream would be resolved with host/port information unless that's meant to be the server's host/port info.

Reply:
single bitrate, multi bitrate.
A provision represents either one single quality stream or multiple variants/views. Those constants represent the line between the two. If a provision level resolves to anything other than zero, its an MBR injest or ABR playback. I am specifying that 1 represents the best quality, 2 then next best, 3 then next, and so on. Those vars are just scaffolding to any API usage for differentiating the live state of a Provision as MBR or not. In a collection of variant streams, MBR_MAIN would be highest.